### DISCUSSION TITLE
Implementing admin access

### DISCUSSION THREAD

**Discussion by amruthcharan** [osiset/laravel-shopify#1256] [General]
As the package uses the user model, How can I implement admin login and access to all shops and other details.

**Comment by Kyon147** [comment]
You could add an additional field to the user DB such as `role` and then just have `store` vs `admin` or `staff` and just create a middleware for routes you only want access to via the `admin` or `staff` role. 

There's a lot of ways you can do it, but that is just one quick way of the top of my head. As I use Nova as well for some basic admin tasks and that is a quick way to distinguish.
